src/main/java/seedu/duke/CommandStructure.java

```diff
@@ -39,6 +39,9 @@ public class CommandStructure {
     public static final String[] CHECK_PROPERTY_FLAGS = {"i/"};
     public static final String[] CHECK_CLIENT_FLAGS = {"i/"};
 
+    public static final int FIRST_INDEX = 0;
```

Comment:
Do consider giving constants a more meaningful name. A good way is to factor in at what context they are utilized.